infra/examples-dev/aws-all/main.tf

```diff
@@ -34,11 +34,15 @@ module "worklytics_connectors" {
   source = "../../modules/worklytics-connectors"
   # source = "git::https://github.com/worklytics/psoxy//infra/modules/worklytics-connectors?ref=rc-v0.4.29"
 
-  enabled_connectors    = var.enabled_connectors
-  example_jira_issue_id = var.example_jira_issue_id
-  jira_cloud_id         = var.jira_cloud_id
-  jira_server_url       = var.jira_server_url
-  salesforce_domain     = var.salesforce_domain
+  enabled_connectors             = var.enabled_connectors
+  example_jira_issue_id          = var.example_jira_issue_id
+  jira_cloud_id                  = var.jira_cloud_id
+  jira_server_url                = var.jira_server_url
+  salesforce_domain              = var.salesforce_domain
+  msft_tenant_id                 = var.msft_tenant_id
+  example_msft_user_guid         = var.example_msft_user_guid
+  google_workspace_example_admin = var.google_workspace_example_admin
+  google_workspace_example_user  = var.google_workspace_example_user
```

Comment:
the google + msft are split out in their own files, to facilitate easy adaption of the example